### ISSUE TITLE
Re-introduce coding standards to documentation

### ISSUE THREAD

**Issue by ganto** [debops/debops#940]
In the "old" multi-repo DebOps documentation we had a page describing a Ansible [coding standards policy](https://github.com/debops/debops-policy/blob/master/docs/code-standards-policy.rst) for DebOps roles. When I'm working with Ansible in my day job I often need something similar and often wanted to reference them as an example. But unfortunately they are not part of the [DebOps documentation](https://docs.debops.org/en/master/debops-policy/index.html) anymore.

What is the reason that they were removed and what needs to be done that something like this is re-added to the DebOps documentation again? I would be willing to work on this again because I think it's a very good way to teach people how (not to) write Ansible code.

**Comment by drybjed** [comment]
They are a part of the repository - kinda. I moved them into the "DebOps Enchancement Proposals" section as [DEP 2 - DebOps code standards](https://docs.debops.org/en/master/dep/dep-0002.html). But it seems that the idea of DEPs didn't really pick up, at least nobody is mentioning it. It again might be my fault - when I write a new role I tend to look how to improve at least one existing coding practice, but then I don't follow up with the documentation explaining it. I guess that I'm waiting for the project codebase to mature before setting things in stone again.

**Comment by htgoebel** [comment]
Shouldn't it this become "DEP 8" for consistency with Python ;-)

**Comment by ganto** [comment]
> They are a part of the repository - kinda. I moved them into the "DebOps Enchancement Proposals" section as DEP 2 - DebOps code standards.

Ok, I see. I was looking at the wrong place then. :confused: 

And the reason why this is a DEP and not part of the policy is because not all Ansible roles comply with the standards yet? So it's an enhancement proposal?

**Comment by drybjed** [comment]
@htgoebel: I thought about DEP 2 being DEP 8 but that felt too much "on the nose". :-)

@ganto: In Python, PEPs are used to define the language as a whole (as I understand it), not just make suggestions. For example when we are talking about code style, when people refer to "PEP 8", everyone understands what they mean by that. I aimed for DEPs to be similar in that they could define different aspects of DebOps in a somewhat formalized manner (the old "policy" documents were converted into DEPs as well, not just the code standards) but I think now that this idea would be relevant when DebOps has more than 100 active developers around so that the community as a whole could be "steered" via DEPs and a consensus on things could be reached that way. As it is right now, they seem to be blown out of proportion. I guess that I went with this idea too soon.
